pkg/test/morerequire/morerequire.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package morerequire includes more require functions than "github.com/stretchr/testify/require"
+// Do not add dependencies on any main code as it will cause cycles.
+package morerequire
```

Comment:
the stuff here is the same as it was, this is just pulled out to help reduce chance of package cycles